toolkit/scss/_previous-next-navigation.scss

```diff
@@ -0,0 +1,127 @@
+/*
+Taken from the GOV.UK component at
+https://github.com/alphagov/static/blob/a9d462e71709d2ff6348bcce7e8c625af2b86114/app/assets/stylesheets/govuk-component/_previous-and-next-navigation.scss
+and
+https://github.com/alphagov/static/blob/da8aeeaa749093eab30286d7fc9f965533b66f47/app/assets/stylesheets/styleguide/_conditionals2.scss
+*/
+@import "_colours.scss";
+@import "_typography.scss";
+
+// Media query helpers. These make producing IE layouts
+// super easy.
+
+// These are desktop and down media queries
+
+// There is also a local version of this in Smartanswers.
+
+$is-ie: false !default;
+
+@mixin media-down($size: false, $max-width: false, $min-width: false) {
```

Comment:
I'd argue for keeping it a verbatim copy-paste. Otherwise bringing in any updates to this pattern means remembering to and figuring out how to convert down to up.

Or, we make a PR against the component and fix it at the source…